src/Manager.php

```diff
@@ -64,11 +64,12 @@ public function postMessage($message, $attachmentPath = null)
      *
      * @var string $photoPath
      */
-    public function postPhoto($photoPath)
+    public function postPhoto($photoPath,$caption=NULL)
```

Comment:
Could you please add space after comma to maintain code style? And also transform `NULL` to lowercase.

Also, you didn't add PHPDoc for new attribute, thats gonna be `@var string|null $caption`